dvc/command/config.py

```diff
@@ -54,10 +68,14 @@ def run(self):
 
         if self.args.value is None and not self.args.unset:
             conf = self.config.load_one(self.args.level)
+            prefix = self._config_file_prefix(
+                self.args.show_origin, self.config, self.args.level
+            )
+
             if remote:
                 conf = conf["remote"]
             self._check(conf, remote, section, opt)
-            logger.info(conf[section][opt])
+            logger.info("{}{}".format(prefix, conf[section][opt]))
```

Comment:
`git config` won't care about `--show-origin` in this scenario, right?

Reply:
I don't see why it would.

Reply:
@mrstegeman So should we care about it?

Reply:
Oh, sorry, I misunderstood your question. Yes, git does use `--show-origin` when `--list` is not provided:

```
$ git config --show-origin branch.master.remote
file:.git/config	origin
```

Reply:
Makes sense. Thanks for clarifying!